model/profile/_meta/fields.yml

```diff
@@ -619,6 +619,15 @@
               count: 1
               description: >
                 Amount of CPU time profiled, in nanoseconds.
+        
+        - name: wall
+          type: group
+          fields:
+            - name: us
```

Comment:
It would be nice to have a single base unit -- on-CPU time is recorded as `profile.cpu.ns`. In theory if we stored in nanoseconds then we could run into overflows when aggregating over a large cluster (hundreds) of service instances, over a significant time period (months to years).

Probably something to consider later, when we firm all of this up.